Xamarin.Forms.Core/Performance.cs

```diff
@@ -1,22 +1,25 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Diagnostics;
 using System.Linq;
 using System.Runtime.CompilerServices;
 using System.Text;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
-	internal static class Performance
+	public static class Performance
```

Comment:
It's in the internals namespace so its not part of our supported public surface.